Workbooks/CosmosDb Fleet/Overview.workbook

```diff
@@ -0,0 +1,2315 @@
+            "isRequired": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.documentdb/fleets": true
+              },
+              "additionalResourceOptions": [
+                "value::1"
+              ],
+              "showDefault": false
+            }
+          },
+          {
+            "id": "57754aaa-671b-412e-84fe-b1258caacf3a",
+            "version": "KqlParameterItem/1.0",
+            "name": "Fleetspaces",
+            "type": 2,
+            "multiSelect": true,
+            "quote": "",
+            "delimiter": ",",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Fleet}/fleetspaces\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2024-12-01-preview\"},{\"key\":\"\",\"value\":\"\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.name\",\"columns\":[]}}]}",
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "showDefault": false
+            },
+            "queryType": 12,
+            "value": [
+              "fleetspace1"
+            ]
+          },
+          {
+            "id": "18f1c50e-63d0-47f4-ab76-b2ebc09bbe49",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resources",
+            "label": "Database Accounts",
+            "type": 5,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "{\"version\":\"ARMEndpoint/1.0\",\"data\":null,\"headers\":[],\"method\":\"GET\",\"path\":\"{Fleet}/fleetspaces/{Fleetspaces}/fleetspaceAccounts\",\"urlParams\":[{\"key\":\"api-version\",\"value\":\"2024-12-01-preview\"}],\"batchDisabled\":false,\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"tablePath\":\"$.value.*.properties.accountResourceIdentifier\",\"columns\":[]}}]}",
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.documentdb/databaseaccounts": true
+              },
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "showDefault": false
+            },
+            "queryType": 12,
+            "value": [
+            ]
+          }
+        ],
+        "style": "formVertical",
+        "queryType": 12
+      },
+      "customWidth": "25",
+      "name": "cosmosDB account parameters",
+      "styleSettings": {
+        "margin": "15px 0 0 0"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "where type  =~ 'Microsoft.DocumentDB/databaseAccounts'\r\n| where name in~ (split(\"{Resources:label}\", \", \"))\r\n| summarize dcount(name)\r\n\r\n",
```

Comment:
This query is doing a count of database accounts, but your parameters are for fleetspace accounts. Is this on purpose? I can see users being confused if parameters did one query but this query does another?

Reply:
So this needs to happen b/c of the arg implementation. We don't have fleetspaceaccount queryable in arg justg yet since this is a preview feature.